conmon/conmon.c

```diff
@@ -632,6 +636,45 @@ int main(int argc, char *argv[])
 	cpid = atoi(contents);
 	ninfo("container PID: %d", cpid);
 
+	/* Setup endpoint for attach */
+	struct sockaddr_un attach_addr = {0};
+	_cleanup_close_ int afd = -1;
+	int asfd = -1;
+
+	if (!exec) {
+		attach_addr.sun_family = AF_UNIX;
+		snprintf(attach_sock_path, PATH_MAX, "/var/run/%s-attach", cid);
+		ninfo("attach sock path: %s", attach_sock_path);
+
+		asfd = open(attach_sock_path, O_CREAT|O_WRONLY);
```

Comment:
Why are you creatting this file only to then delete it?
Can't you just use the bind() call to create it?